code/_onclick/hud/fullscreen.dm

```diff
@@ -156,9 +156,38 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
 	icon_state = "noise"
 
 /atom/movable/screen/fullscreen/high
-	icon = 'icons/hud/screen_gen.dmi'
-	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "druggy"
+	alpha = 255
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_ABOVE_ALL + 1 //Infinity plus one (not actually)
+	blend_mode = BLEND_MULTIPLY
+
+/atom/movable/screen/fullscreen/high/Initialize(mapload)
+
+	. = ..()
+
+	alpha = 0
+
+	animate(
+		src,
+		alpha = 255,
+		time = 10 SECONDS
+	)
```

Comment:
this seems a little silly, what happens if it's created before being seen by the client?